.github/workflows/test.yml

```diff
@@ -17,10 +17,11 @@ jobs:
       matrix:
           os: [ubuntu-18.04]
     steps:
-    - uses: ros-tooling/setup-ros@0.0.13
-    - uses: ros-tooling/action-ros-ci@0.0.13
+    - uses: ros-tooling/setup-ros@0.0.25
+    - uses: ros-tooling/action-ros-ci@0.0.18
```

Comment:
I'll suggest that you stick to the same versions as master here for consistency (`setup-ros@0.0.23` and `action-ros-ci@0.0.17`).

If we want to upgrade further, we should probably do it on master first and then backport to Foxy.

Reply:
Updated: https://github.com/ros2/rcpputils/pull/104/commits/1860573f9d93c60cf3e484a5972709337fb442e9

We should probably remove the Github workflow in master (first), right ? and backport the change.

Reply:
> We should probably remove the Github workflow in master (first), right ? and backport the change.

Yeah, that would be my suggestion.